shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -63,10 +63,10 @@ SkRect AndroidExternalViewEmbedder::GetViewRect(int view_id) const {
   const EmbeddedViewParams& params = view_params_.at(view_id);
   // TODO(egarciad): The rect should be computed from the mutator stack.
```

Comment:
nit: this `TODO` can now be removed

Reply:
We should keep this as the bounding rect doesn't take clipping into consideration yet.